src/sidebar.ts

```diff
@@ -17,7 +17,7 @@ const homeLink: SidebarItem = {
 
 const openIdSSO: SidebarItemConfig = {
   type: "category",
-  label: "OpenID Connect SSO",
+  label: "OAuth2 client",
```

Comment:
maybe it's just me but I find this very unclear / have no idea what is in this category before I open it. 

Wouldn't OIDC be the thing that everyone knows here?